bitcraze_crazyflie_2/cf2.xml

```diff
@@ -0,0 +1,119 @@
+<mujoco model="cf2">
```

Comment:
Add newlines between the different sections of the XML (e.g., between option and compiler and default etc.).